src/ibex_bluesky_core/plan_stubs/__init__.py

```diff
@@ -1 +1,42 @@
 """Core plan stubs."""
+
+from typing import Callable, Generator, ParamSpec, TypeVar, cast
+
+import bluesky.plan_stubs as bps
+from bluesky.utils import Msg
+
+P = ParamSpec("P")
+T = TypeVar("T")
+
+
+CALL_SYNC_MSG_KEY = "ibex_bluesky_core_call_sync"
+
+
+def call_sync(func: Callable[P, T], *args: P.args, **kwargs: P.kwargs) -> Generator[Msg, None, T]:
+    """Call a synchronous user function in a plan, and returns the result of that call.
+
+    Attempts to guard against the most common pitfalls of naive implementations, for example:
```

Comment:
```suggestion
    Attempts to guard against the most common pitfalls of naive implementations, for example:
    
```
needed by MyST to render the bullet points